src/pages/jiyeon/Main/MainJiyeon.js

```diff
@@ -1,7 +1,293 @@
+                alt="bookmark"
+              />
+              <span className="profileBoxText">저장됨</span>
+            </div>
+            <div className="profileBoxInnerBox">
+              <img
+                src="/images/jiyeon/settings_FILL0_wght400_GRAD0_opsz48.png"
+                alt="setting"
+              />
+              <span className="profileBoxText">설정</span>
+            </div>
+            <div className="profileBoxInnerBox">
+              <span className="profileBoxText">로그아웃</span>
+            </div>
+          </div>
+        </div>
+
+        <div id="main">
+          <div className="feeds">
+            <article class="articles">
+              <div id="feedTop">
+                <div id="personal">
+                  <img
+                    id="personalProfile"
+                    src="/images/jiyeon/KakaoTalk_20230508_214810406.jpg"
+                    alt="personalProfile"
+                  />
+                  <span id="personalId">yeonnn</span>
+                </div>
+                <img
+                  src="/images/jiyeon/more_horiz_FILL0_wght400_GRAD0_opsz48.png"
+                  alt="more"
+                />
+              </div>
+              <img src="/images/jiyeon/cat.jpg" alt="cat" />
+              <div id="feedBottom">
+                <div id="feedBottomLeft">
+                  <img src="/images/jiyeon/heart.png" alt="heart" />
+                  <img src="/images/jiyeon/chat.png" alt="comment" />
+                  <img src="/images/jiyeon/upload.png" alt="share" />
+                </div>
+                <img src="/images/jiyeon/ribbon.png" alt="bookmark" />
+              </div>
+              <div id="commentBox">
+                <div id="howMuchLike">
+                  <img
+                    id="likeProfile"
+                    src="/images/jiyeon/KakaoTalk_20230508_214810406.jpg"
+                    alt="likeProfile"
+                  />
+
+                  <span>
+                    <span className="bold">aineworld</span> 님
+                    <span className="bold">외 10명</span>이 좋아합니다
+                  </span>
+                </div>
+                <div class="comment">
+                  <span className="bold">yeonnn</span> 세상에서 제일 귀여운 지구
+                  <span className="grey">...더 보기</span>
+                  <div className="grey">댓글 127개 모두 보기</div>
+                </div>
+                <div id="existComment">
+                  <span className="bold">neceosecius</span>
+                  <span>귀여워💕</span>
+                  <button className="deleteBtn">삭제</button>
+                  <img
+                    className="heartIcon"
+                    src="/images/jiyeon/heart.png"
+                    alt="comment"
+                  />
+                </div>
+                {comment.map(function (a, i) {
+                  return <WriteNewComment comment={comment} index={i} />;
+                })}
```

Comment:
```suggestion
                {comment.map(function (a, i) {
                  return <WriteNewComment comment={a} index={i} />;
                })}
```

map 메서드는 배열에서 각각의 요소에 접근해서 새로운 배열을 반환하는 메서드입니다!
여기서 comment는 하나의 요소가 아닌 전체 배열을 주고 있어서
위에 컴포넌트 분리한 부분에서도 index 값을 찾아가는 방식으로 적용되고 있는데
위에 변경해 드린 부분 처럼 a 를 comment로 넘겨줘야 합니다!